### ISSUE TITLE
Equation editor

### ISSUE THREAD

**Issue by TheMonsterFromTheDeep** [TheMonsterFromTheDeep/termcalc#7]
Basic equations are implemented, but there is at least one big problem:

Graphs cannot be removed once added.

This, however, is only one thing that could be fixed with a dedicated equation editor - it should also be easy to update equations and see their graphs change in real-time, and it should also be possible to change the color of a graph and maybe make it graph dotted or with thicker lines, as well as hide it.

Another thing I want to do with an equation editor is make equations rendered in a LaTeX-type style using unicode characters such as superscripts and symbols like π.

To put it simply, I would like to make it act like [Desmos](https://www.desmos.com/).
